### PR TITLE
Remove MetaClient.Ping from `/ping` and move it to `/status`

### PR DESCRIPTION
**Pull request by toddboom**

### CODE REVIEW

**Comment by jwilder** [comment]
:+1:

File: services/httpd/handler.go [hunk 1]
```diff
@@ -567,6 +575,12 @@ func (h *Handler) serveOptions(w http.ResponseWriter, r *http.Request) {
 // servePing returns a simple response to let the client know the server is running.
 func (h *Handler) servePing(w http.ResponseWriter, r *http.Request) {
 	h.statMap.Add(statPingRequest, 1)
+	w.WriteHeader(http.StatusNoContent)
+}
+
+// serveStatus returns a simple response to let the client know the whole cluster is running.
+func (h *Handler) serveStatus(w http.ResponseWriter, r *http.Request) {
+	h.statMap.Add(statPingRequest, 1)
```
Comment (corylanou) [inline services/httpd/handler.go]: nit: This should probably be a different constant. `statStatusRequest` perhaps?

**Comment by corylanou** [comment]
minor nit, otherwise +1

**Comment by toddboom** [comment]
@corylanou good call, fixed - thanks!

**Pull request merged**